client/src/style/index.pcss

```diff
@@ -9,3 +9,23 @@
 .squids-list__tile {
   @apply my-3 text-lg;
 }
+
+.squids-pagination {
+  @apply text-center text-xl mt-10;
+}
+
+.squids-pagination__scroll-icon {
```

Comment:
I'll usually nest elements within their respective blocks for specificity and organization.